src/plugins/timelist/Timelist.vue

```diff
@@ -44,47 +44,45 @@ const SCROLL_TIMEOUT = 10000;
 const ROW_HEIGHT = 30;
 const TIME_FORMAT = 'YYYY-MM-DD HH:mm:ss:SSS';
 const headerItems = [
-  {
-    defaultDirection: true,
-    isSortable: true,
-    property: 'start',
-    name: 'Start Time',
-    format: function (value, object, key, openmct) {
-      const clock = openmct.time.clock();
-      if (clock && clock.formatTime) {
-        return clock.formatTime(value);
-      } else {
-        return `${moment(value).format(TIME_FORMAT)}Z`;
-      }
-    }
-  },
-  {
-    defaultDirection: true,
-    isSortable: true,
-    property: 'end',
-    name: 'End Time',
-    format: function (value, object, key, openmct) {
-      const clock = openmct.time.clock();
-      if (clock && clock.formatTime) {
-        return clock.formatTime(value);
-      } else {
-        return `${moment(value).format(TIME_FORMAT)}Z`;
-      }
-    }
-  },
-  {
-    defaultDirection: false,
-    property: 'duration',
-    name: 'Time To/From',
-    format: function (value) {
-      let result;
-      if (value < 0) {
-        result = `-${getPreciseDuration(Math.abs(value))}`;
-      } else if (value > 0) {
-        result = `+${getPreciseDuration(value)}`;
-      } else {
-        result = 'Now';
-      }
+    {
+        defaultDirection: true,
+        isSortable: true,
+        property: 'start',
+        name: 'Start Time',
+        format: function (value, object, key, openmct) {
+            const clock = openmct.time.clock();
+            if (clock && clock.formatTime) {
+                return clock.formatTime(value);
+            } else {
+                return `${moment.utc(value).format(TIME_FORMAT)}Z`;
+            }
+        }
+    }, {
+        defaultDirection: true,
+        isSortable: true,
+        property: 'end',
+        name: 'End Time',
+        format: function (value, object, key, openmct) {
+            const clock = openmct.time.clock();
+            if (clock && clock.formatTime) {
+                return clock.formatTime(value);
```

Comment:
This is not part of the Time API. It appears to have been implemented on `RemoteClock` for internal use. The correct way to do this is:
```
const timeFormat = openmct.time.timeSystem().timeFormat;
const timeFormatter = openmct.telemetry.getFormatter(timeFormat);
```
This will work in both realtime and fixed time modes.

Reply:
Excellent suggestion, this certainly reduces complexity in special logic for fixed vs realtime modes and the end result is more readable.